parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -3436,8 +3438,6 @@ static void jdf_generate_internal_init(const jdf_t *jdf, const jdf_function_entr
         coutput("  int32_t nb_tasks = 0, saved_nb_tasks = 0;\n");
         /* prepare the epilog output to prevent compiler from complaining about initialized but unused data */
         string_arena_add_string(sa_end, "(void)saved_nb_tasks;\n");
-    } else {
-        coutput("  int32_t nb_tasks = 0;\n");
```

Comment:
This is removed but is still used later in the generated code, was it a duplicate alias of the same variable?

Reply:
I still have a
```   if(need_to_count_tasks) {
        coutput("  int32_t nb_tasks = 0, saved_nb_tasks = 0;\n");
```
at line 3437 of the new code